testing/regress/ecl/parquetDecimals.ecl

```diff
@@ -0,0 +1,55 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+//class=parquet
+//class=embedded
+
+IMPORT PARQUET;
+
+layout := RECORD
+    DECIMAL d1;
+    DECIMAL1_0 d2;
+    DECIMAL16_8 d3;
+    DECIMAL32_16 d4;
+    UDECIMAL64_32 d5;
+    DECIMAL64_32 d6;
+END;
+
+decimal_data := DATASET([{(DECIMAL) '0.12345678901234567890123456789012',
+                          (DECIMAL1_0) '1',
+                          (DECIMAL16_8) '12345678.12345678',
+                          (DECIMAL32_16) '1234567890123456.1234567890123456',
+                          (UDECIMAL64_32) '12345678901234567890123456789012.12345678901234567890123456789012',
+                          (DECIMAL64_32) '-12345678901234567890123456789012.12345678901234567890123456789012'
+                          }], layout);
+
+overwriteOption := TRUE;
+ParquetIO.Write(decimal_data, '/home/jack/dev/debug/etc/HPCCSystems/mydropzone/regress/parquet/decimal.parquet', overwriteOption);
```

Comment:
This does not pass the smoketest because it is a reference to a file path that does not exist.  This probably needs to be hthor only and use a local relative filename.

Reply:
Fixed test file tags to exclude it from running on Thor and Roxie. 

The Arrow library does not support creating arrow::fs::Filesystems with relative paths. It would be possible to add support if the plugin was able to get the path context from somewhere else and add it to the relative path, but for now I have set the path where the test files are created to '/etc/HPCCSystems/mydropzone/regress/decimal.parquet'. Is this appropriate or would a more general location like '/tmp/decimal.parquet' be better?